packages/docs/src/routes/docs/components/events/index.mdx

```diff
@@ -132,7 +132,7 @@ const CmpButton = component$((props: CmpButtonProps) => {
 
 Notice that we can pass the `props.onClick$` method directly to the `onDblclick$` attribute as seen on `<button>` element. (see attribute `onDblclick$={props.onClick$}`) This is because both the inputting prop `onClick$` as well as JSX prop `onDblclick` are of type `PropFunction<?>` (and both have `$` suffix).
 
-However, it is not possible to pass `props.onClick$` to `onClick$` because the types don't match. (This would result in the type error: `onClick$={props.onClick$}`) Instead, the `$` is reserved for inlined closures. In our example, we would like to `console.log("clicked")` after we process the `props.onClick` callback. We can do so with the `props.onClick$()` method. This 1) lazy-loads the code, 2) restores the closure state, and 3) invokes the closure. The operation is asynchronous and therefore returns a promise, which we can resolve using the `await` statement.
+However, it is not possible to pass `props.onClick$` to `onClick$` because the types don't match. (This would result in the type error: `onClick$={props.onClick$}`) Instead, the `$` is reserved for inlined closures. In our example, we would like to `console.log("clicked")` after we process the `props.onClick` callback. We can do so with the `props.onClick$()` method. This will lazy-loads the code, then restores the closure state, and finally invokes the closure. The operation is asynchronous and therefore returns a promise, which we can resolve using the `await` statement.
```

Comment:
Thanks!

These fixes require a bit of a touch up :)

"This will lazy-**load**.." (instead of "lazy-loads")

And this comment apply to "then restore" and "finally, will invoke.."

We can also lay it out as follows:

This method will:

1) Lazy load the code
2) Restore the closure state
3) Invoke the closure

But I don't have a strong opinion about which style to choose here

WDYT?

Reply:
Yeah you are right, this more easily to read instead of inline numbering.

Reply:
awesome!

can you push a lovely new commit with this change please? 😊🙏

Thanks

Reply:
I have made the changes

Reply:
Awesome thanks so much!

Reply:
In [rendering docs](https://qwik.builder.io/docs/components/rendering/) their is inline numbering. Shall i change it?

WDYT?

Reply:
Yes, thanks